PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+## Description
+
```

Comment:
Might be good to put some prompts in here, maybe something like: 
What does this PR add/fix/remove?
What should reviewers pay special attention to?
Any breaking changes?

Reply:
Done